modules/microshift-install-bootc-build-image.adoc

```diff
@@ -71,19 +72,36 @@ RUN systemctl enable microshift-make-rshared.service
 Podman uses the host subscription information and repositories inside the container when building the container image. If the `rhocp` and `fast-datapath` repositories are not available on the host, the build fails.
 ====
 
-. Create a local bootc image by running the following image build command:
+. Set the `PULL_SECRET` environment variable:
 +
-[source,terminal,subs="+quotes"]
+[source,terminal]
 ----
 PULL_SECRET=~/.pull-secret.json
-USER_PASSWD=_<your-redhat-user-password>_ # <1>
+----
+
+. Configure the `USER_PASSWD` environment variable:
++
+[source,terminal,subs="+quotes"]
+----
+USER_PASSWD=_<redhat_user_password>_ <1>
+----
+<1> Replace _<redhat_user_password>_ with your password.
+
+. Configure the `IMAGE_NAME` environment variable:
++
+[source,terminal,subs="attributes+"]
+----
 IMAGE_NAME=microshift-{product-version}-bootc
+----
```

Comment:
This is not rendered properly

Reply:
I am seeing Branch Build here, which is as expected. Maybe we could hop on a quick call Monday if you're not seeing what I am, figure it out?

<img width="819" alt="Screenshot 2025-03-20 at 12 46 24 PM" src="https://github.com/user-attachments/assets/516de633-ebe0-4c57-b345-52dde0828785" />